<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-color: rgb(189, 189, 189);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>122</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color:#20B2AA;
border-width: 1px;
border-color: #20B2AA;
border-style: solid;
border-radius: 7;
padding: 4px;
font-size: 13px;
padding-left: -10px;
padding-right: -10px;
min-width: 140px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
    </property>
    <property name="text">
     <string>afficher </string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="ajouter_p">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>641</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#ajouter_p{
background-color:transparent;
border-radius:10px;
border: 1px solid #20B2AA;
color:#20B2AA;
}</string>
    </property>
    <property name="title">
     <string>Affichage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>122</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color:#20B2AA;
border-width: 1px;
border-color: #20B2AA;
border-style: solid;
border-radius: 7;
padding: 4px;
font-size: 13px;
padding-left: -10px;
padding-right: -10px;
min-width: 140px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>122</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color:#20B2AA;
border-width: 1px;
border-color: #20B2AA;
border-style: solid;
border-radius: 7;
padding: 4px;
font-size: 13px;
padding-left: -10px;
padding-right: -10px;
min-width: 140px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
background-color:transparent;
border-radius:10px;
border: 1px solid #20B2AA;
color:#20B2AA;
}</string>
    </property>
    <property name="title">
     <string>Formulaire</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>libelle</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_libelle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_quantite">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="vvv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>quantite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>dateAjout</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_localisation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>localisation</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_dateAjouter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>122</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color:#20B2AA;
border-width: 1px;
border-color: #20B2AA;
border-style: solid;
border-radius: 7;
padding: 4px;
font-size: 13px;
padding-left: -10px;
padding-right: -10px;
min-width: 140px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="le_categorie">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>e</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Categorie</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-24</x>
      <y>-7</y>
      <width>1201</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resources/v870-mynt-13.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>ajouter_p</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>tableView</zorder>
   <zorder>modifier</zorder>
   <zorder>supprimer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
